absl/time/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright 2017 The Abseil Authors.
+#
+# Licensed under the Apache License Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+list(APPEND TIME_PUBLIC_HEADERS
+  "clock.h"
+  "time.h"
+)
+
+
+list(APPEND TIME_INTERNAL_HEADERS
+  "internal/test_util.h"
+)
+
+list(APPEND TIME_SRC
+    "time.cc"
+    "clock.cc"
+    "duration.cc"
+    "format.cc"
+    ${TIME_PUBLIC_HEADERS}
+    ${TIME_INTERNAL_HEADERS}
+)
+set(TIME_PUBLIC_LIBRARIES base stacktrace int128 ${CCTZ_LIBRARIES})
+
+abseil_library(
+  TARGET
+      time
+  SOURCES
+      ${TIME_SRC}
+  PUBLIC_LIBRARIES
+      ${TIME_PUBLIC_LIBRARIES}
+)
+add_dependencies(time ${ABSEIL_PRE_DEPENDENCIES})
```

Comment:
This should not be necessary, feels ugly.
It's here just because cctz, when bundled might end up here?
The external project stuff leaking here is not nice.

Reply:
I do not really care how it feels, but I care that all Abseil dependencies are built before anything else :)